test/gui/shared/steps/server_context.py

```diff
@@ -112,3 +112,16 @@ def step(context, user_name, resource_name):
     test.compare(
         has_link_share, False, f"Resource '{resource_name}' have public link share"
     )
+
+
+@Then(
+    r'as user "([^"].*)" folder "([^"].*)" should contain "([^"].*)" items in the server',
+    regexp=True,
+)
+def step(context, user, folder_name, item_number):
```

Comment:
```suggestion
def step(context, user_name, folder_name, item_number):
```
Consistent naming throughout the implementation.